### ISSUE TITLE
wallet missing after upgrading from 1.53 to  v2.0.0-rc.4

### ISSUE THREAD

**Issue by diomark** [alephium/desktop-wallet#654]
I had alephium desktop wallet 1.5x version running and opened on one wallet
Re2.0.0rc4 installer - it asked to close the alephium desktop wallet and upgrade it
upgraded, came up. the wallet that was previously open was missing from list of wallets to open
tested logging into a different wallet/ log out, same list. (missing the one)
closed desktop wallet, re-opened it, list included the missing wallet now.

OS is Windows 10 btw

**Comment by nop33** [comment]
Thanks a lot for the report, I'll investigate!

**Comment by diomark** [comment]
in case it matters; I had 5 wallets today. 4 displayed the one that was prevously open didn't.

**Comment by nop33** [comment]
Not sure I understand your last message. The problem was fixed after the app was closed and then re-opened, right?

**Comment by diomark** [comment]
correct.. after fully closing the wallet/exiting then restarting it it properly showed all wallets.

**Comment by nop33** [comment]
OK great, it must be some issue with the app state, I'll work on it, great feedback, thanks 💯

**Comment by nop33** [comment]
Hey @diomark, I tried many times but I couldn't reproduce the problem. I've installed 1.5.3 in a fresh Windows machine, created 2 wallets, downloaded 2.0.0-rc.4 and tried to install it while the 1.5.3 was still running. I clicked "OK" to close the currently running 1.5.3 app and when the installation finished, both of my wallets were there. I tested with macOS as well and also no problems.

Since we don't expect users to install our wallet manually anymore (remember, we have the automatic update system), I expect that this will not be a problem that reproduces itself.

Let's keep this issue open in case someone else faces the same problem.
